frontend/src/assets/i18n/en.json

```
@@ -45,11 +45,45 @@
     "telephonePlaceholder": "Phone Number",
     "document-type-label": "Document Type",
     "documentTypePlaceholder": "Select Document Type",
+    "search-document-types": "Search document types...",
```

Biome (2.1.2)

[error] The key search-document-types was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)